<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_imeds_data</class>
 <widget class="QDialog" name="add_imeds_data">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add IMEDS Data</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>431</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_filename">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_filebrowse">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_seriesname">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>40</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Label:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="text_seriesname">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Series 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_color">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>70</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Select Color:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_IMEDSColor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_filename">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>10</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Filename:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>add_imeds_data</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>add_imeds_data</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
